src/role_quester.js

```diff
@@ -33,7 +33,7 @@ roles.quester.questWon = function(creep, quest) {
   };
   creep.room.terminal.send(RESOURCE_ENERGY, 100, quest.player.room, JSON.stringify(response));
   delete Memory.quests[creep.memory.level];
-  creep.suicide();
+  Creep.recycleCreep(creep);
```

Comment:
same here timeToTravel vs born

Reply:
added the judge into recycleCreep function in